corehq/ex-submodules/casexml/apps/case/tests/test_rebuild.py

```diff
@@ -109,7 +109,7 @@ def test_couch_soft_rebuild(self):
         self.assertEqual(case.p3, 'p3-2') # new
         self.assertEqual(4, len(case.actions)) # create + update (2 actions) + 2 updates
         a0 = case.actions[1]
-        self.assertEqual(a0.updated_known_properties['opened_on'], case.opened_on)
+        self.assertEqual(a0.updated_known_properties['opened_on'], case.opened_on.date())
```

Comment:
i'm actually confused by this - shouldn't the update / save still save a datetime? i thought we only converted to dates during restore